### PR TITLE
Update enrollment group documentation

### PR DESCRIPTION
**Pull request by utsavized**
Documentation text update to enrollment group certificate verification command.

- [x] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

### CODE REVIEW

File: azext_iot/central/services/enrollment_group.py [hunk 1]
```diff
@@ -418,6 +418,14 @@ def verify_x509(
     Verify the primary or secondary x509 certificate of an enrollment group
     by providing a certificate with the signed verification code.
 
+    Please note that if an enrollment group was created with an unverified x509
```
Comment (digimaun) [inline azext_iot/central/services/enrollment_group.py]: If this is intended to be user facing reference doc updates you'll want to update the description in the help module here https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/central/_help.py#L1759

Comment (utsavized) [reply azext_iot/central/services/enrollment_group.py]: @digimaun my intention is to update this http public doc: https://learn.microsoft.com/en-us/cli/azure/iot/central/enrollment-group?view=azure-cli-latest#az-iot-central-enrollment-group-verify-certificate

Is that pulling from _help.py? I was under the assumption that the _help.py was for the cli help command.

Comment (vilit1) [reply azext_iot/central/services/enrollment_group.py]: yes the file paymaun linked is the correct help file you want to edit. What you are doing here is just poking a doc string that affects nothing (beyond what a dev would read when they stumble here)

Comment (utsavized) [reply azext_iot/central/services/enrollment_group.py]: @vilit1 @digimaun Great, thanks for pointing out the right location. Will push an update soon.

**Comment by utsavized** [comment]
> 

@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by utsavized** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by utsavized** [comment]
@digimaun @vilit1 Updated, thanks!

**Review by digimaun — APPROVED** [review]

**Pull request merged**